frontend/src/components/dialogs/CreateFolderDialog.tsx

```
@@ -0,0 +1,194 @@
+              className="text-gray-400 hover:text-gray-600 dark:hover:text-gray-200 transition-colors"
+            >
+              <span className="material-symbols-outlined">close</span>
+            </button>
+          </div>
+
+          <div className="space-y-6">
+            {/* 1. 폴더 이름 입력 섹션 */}
+            <div>
+              <div className="flex items-center gap-2 mb-2">
+                <span className="w-1.5 h-1.5 rounded-full bg-purple-500"></span>
+                <label className="text-sm font-semibold text-gray-700 dark:text-gray-200">Folder Name</label>
```

Biome (2.4.4)

[error] A form label must be associated with an input.

(lint/a11y/noLabelWithoutControl)